### ISSUE TITLE
Drag and drop on page assets while editing

### ISSUE THREAD

**Issue by ababushkin** [Ladoo/MyMatrix#72]
A common task that an author has to carry out, is editing pages. While doing so, they go through a variety of tasks, such as:
- Changing asset statuses;
- Editing metadata;
- Editing assets;
- Editing attachments (such as images and documents) to the content page;
- Adding new attachments;
- Saving and Previewing.

We so far have tackled 2 major items. Adding new attachments and saving. This feature proposes to improve upon editing & adding new attachments, so that it can be done in context of the asset they're editing (admin interface of course).

In summary, the enhancements would work like so:
- A user is editing a page and would like to upload an image, a pdf and an excel spreadsheet;
- They select these 3 assets, and drag and drop them over the page they're currently editing;
- myMatrix automatically recognises this and uploads them as children assets.
